Appirater.m

```diff
@@ -46,7 +46,14 @@ NSString *const kAppiraterRatedCurrentVersion		= @"kAppiraterRatedCurrentVersion
 NSString *const kAppiraterDeclinedToRate			= @"kAppiraterDeclinedToRate";
 NSString *const kAppiraterReminderRequestDate		= @"kAppiraterReminderRequestDate";
 
+#ifdef __IPHONE_OS_VERSION_MAX_ALLOWED
 NSString *templateReviewURL = @"itms-apps://itunes.apple.com/WebObjects/MZStore.woa/wa/viewContentsUserReviews?id=APP_ID&onlyLatestVersion=true&pageNumber=0&sortOrdering=1&type=Purple+Software";
+#else
+//annoyingly this seems to open in iTunes rather than the mac app store - couldn't find a way to deeplink to the mac app review panel
+//NSString *templateReviewURL = @"https://userpub.itunes.apple.com/WebObjects/MZUserPublishing.woa/wa/addUserReview?id=APP_ID&onlyLatestVersion=true&type=Purple+Software";
+NSString *templateReviewURL = @"http://itunes.apple.com/us/app/rainbow-blocks/idAPP_ID?mt=12&ls=1";
```

Comment:
I used macappstore://itunes.apple.com/app/id{APP_ID_HERE}?mt=12 and it works without opening safari. All you need is the app id